twister2/builder/west.py

```diff
@@ -42,8 +44,9 @@ def build(self, platform: str, build_dir: str | Path | None = None, **kwargs) ->
                 command,
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
-                cwd=self.zephyr_base.resolve(),
-                env=self.env,
+                # TODO: below lines are causing failures, need proper investigation
+                #cwd=self.zephyr_base.resolve(),
```

Comment:
Please keep one space after # and before beginning of a comment (as it is in line 47). This will be reported as a warning by flake8. 
Maybe you should use FIXME instead of TODO.